setup_pocketsphinx.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+
+
+echo "##### This script installs its libraries into system folders, so please make sure you are running it as a user with proper permissions."
+echo "##### Hit any key to continue..."
+
+read ok
+echo "##### Installing pocketsphinx...."
+rm -rf pocketsphinx
+mkdir pocketsphinx
+cd pocketsphinx
+git clone https://github.com/mozilla/sphinxbase
+git clone https://github.com/mozilla/pocketsphinx
+cd sphinxbase
+./configure && make && make install 
+cd ../pocketsphinx
+./configure && make && make install 
```

Comment:
How is this supposed to work if you are not root.

You should likely use something like...
```
./configure --prefix=/somewhere/else/than/usr/local
```